src/accessibility.cs

```diff
@@ -406,4 +406,229 @@ enum AXTechnology {
 		[Field ("AXTechnologyZoom")]
 		Zoom,
 	}
+
+	[iOS (18, 2), NoTV, NoMacCatalyst, NoMac]
+	[Native]
+	public enum AXFeatureOverrideSessionError : long {
+		Undefined = 0,
+		AppNotEntitled,
+		OverrideIsAlreadyActive,
+		OverrideNotFoundForUUID,
+	}
+
+	[iOS (18, 2), NoTV, NoMacCatalyst, NoMac]
+	[Flags]
+	[Native]
+	public enum AXFeatureOverrideSessionOptions : ulong {
+		Grayscale = 1uL << 0,
+		InvertColors = 1uL << 1,
+		VoiceControl = 1uL << 2,
+		VoiceOver = 1uL << 3,
+		Zoom = 1uL << 4,
+	}
+
+	[iOS (18, 2), NoTV, NoMacCatalyst, NoMac]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface AXFeatureOverrideSession {
+	}
+
+	[iOS (18, 2), NoTV, NoMacCatalyst, NoMac]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface AXFeatureOverrideSessionManager {
+		[Static]
+		[Export ("sharedInstance")]
+		AXFeatureOverrideSessionManager SharedInstance { get; }
+
+		[Export ("beginOverrideSessionEnablingOptions:disablingOptions:error:")]
+		[return: NullAllowed]
+		AXFeatureOverrideSession BeginOverrideSession (AXFeatureOverrideSessionOptions enableOptions, AXFeatureOverrideSessionOptions disableOptions, [NullAllowed] out NSError error);
+
+		[Export ("endOverrideSession:error:")]
+		bool EndOverrideSession (AXFeatureOverrideSession session, [NullAllowed] out NSError error);
+	}
+
+	[TV (18, 2), Mac (15, 2), iOS (18, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (NSObject))]
+	interface AXMathExpression {
+	}
+
+	[TV (18, 2), Mac (15, 2), iOS (18, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (AXMathExpression))]
+	interface AXMathExpressionNumber {
```

Comment:
This has a ctor and properties are readonly

```suggestion
	[BaseType (typeof (AXMathExpression))]
	[DisableDefaultCtor]
	interface AXMathExpressionNumber {
```